snap/local/snap-setup-mod/Init.py

```diff
@@ -0,0 +1,10 @@
+import os
+import sys
+
+pythonpath = os.environ.get("SNAP_PYTHONPATH")
+
+if pythonpath:
+  print(f"Adding snap-specific PYTHONPATH to sys.path: {pythonpath}")
+  os.environ["PYTHONPATH"] = pythonpath
+  for path in pythonpath.split(":"):
+    sys.path.insert(0, path)
```

Comment:
This injects (prepends) the snap's idiosyncratic PYTHONPATH into FreeCAD. It's necessary for pip and can't work with appended paths as provided by the --python-path option.